pkg/collect/mysql.go

```diff
@@ -15,18 +15,43 @@ func Mysql(c *Collector, databaseCollector *troubleshootv1beta2.Database) (Colle
 	databaseConnection := DatabaseConnection{}
 
 	db, err := sql.Open("mysql", databaseCollector.URI)
+	defer db.Close()
 	if err != nil {
 		databaseConnection.Error = err.Error()
 	} else {
 		query := `select version()`
 		row := db.QueryRow(query)
+
 		version := ""
 		if err := row.Scan(&version); err != nil {
 			databaseConnection.Error = err.Error()
 		} else {
 			databaseConnection.IsConnected = true
 			databaseConnection.Version = version
 		}
+
+		rows, err := db.Query("SHOW VARIABLES")
+		defer rows.Close()
+
+		if err != nil {
+			databaseConnection.Error = err.Error()
+		}
+
+		if err != nil {
```

Comment:
Line 36 is already doing this check

Reply:
Oh yep, cleaned up some code here and must have left that one in there. Removing it now.